sass/layout/_markdown.scss

```diff
@@ -111,6 +110,38 @@ h6 {
     .hljs {
         position: relative;
         visibility: visible;
+        white-space: pre;
+        display: flex;
+        flex-direction: column;
+
+        >div {
+            display: flex;
+        }
+
+        /* for block of numbers */
+        .hljs-ln-numbers {
+            -webkit-touch-callout: none;
+            -webkit-user-select: none;
+            -khtml-user-select: none;
+            -moz-user-select: none;
+            -ms-user-select: none;
+            user-select: none;
+            color: #ccc;
+            border-right: 1px solid #CCC;
```

Comment:
I think, colors should use values from the theme otherwise users with custom themes might not see them properly
not sure which ones, though... @esethna any thoughts on this?

Reply:
Hey @Willyfrog, not sure about this because the color for border is also not coming from any theme.